src/pybind/ceph_argparse.py

```diff
@@ -1347,4 +1347,7 @@ def json_command(cluster, target=('mon', ''), prefix=None, argdict=None,
         else:
             raise
 
+    if ret == -errno.EINVAL:
+        raise RuntimeError('"{0}": bad command or argument: {1}'.format(
```

Comment:
Here is the problem. `return json_command("foo")` will get 0 when json_command raises exception here!

Maybe we shouldn't panic here, how about putting error message to `outs` only.